src/catalog/src/kvbackend/manager.rs

```diff
@@ -57,6 +55,7 @@ pub struct KvBackendCatalogManager {
     // Now we use `NoopKvCacheInvalidator` for Standalone mode. In Standalone mode, the KV backend
     // is implemented by RaftEngine. Maybe we need a cache for it?
     backend_cache_invalidator: KvCacheInvalidatorRef,
+    table_metadata_cache_invalidator: TableMetadataCacheInvalidator,
```

Comment:
"It's strange that `backend_cache_invalidator` and `table_metadata_cache_invalidator` are together, shouldn't `backend_cache_invalidator` be inside `table_metadata_cache_invalidator`?"